.github/workflows/causal_lm_cpp.yml

```diff
@@ -374,6 +374,7 @@ jobs:
           python -m pip install --upgrade-strategy eager "optimum>=1.14" -r ./llm_bench/python/requirements.txt "transformers<4.38" ./thirdparty/openvino_tokenizers/[transformers] --extra-index-url https://download.pytorch.org/whl/cpu
           python ./llm_bench/python/convert.py --model_id TinyLlama/TinyLlama-1.1B-Chat-v1.0 --output_dir ./TinyLlama-1.1B-Chat-v1.0/ --precision FP16
           convert_tokenizer ./TinyLlama-1.1B-Chat-v1.0/pytorch/dldt/FP16/ --output ./TinyLlama-1.1B-Chat-v1.0/pytorch/dldt/FP16/ --with-detokenizer
+          apt install libtbb-dev
```

Comment:
```suggestion
          sudo apt install libtbb-dev
```
![image](https://github.com/openvinotoolkit/openvino.genai/assets/2566854/e8c3c015-03de-449b-8509-f5a6b43d1acc)

All other jobs are failed, because common cmake is used for all of them and TBB is required everywhere.